.install/amazon_linux_2.sh

```diff
@@ -6,6 +6,10 @@ export DEBIAN_FRONTEND=noninteractive
 $MODE yum update -y
 $MODE yum groupinstall -y "Development Tools"
 $MODE yum remove -y gcc # remove gcc 7
-$MODE yum install -y wget git openssl-devel openssl gcc10 gcc10-c++
+$MODE yum install -y wget git openssl11 openssl11-devel gcc10 gcc10-c++
 $MODE update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc10-gcc 60 --slave /usr/bin/g++ g++ /usr/bin/gcc10-g++
+# make openssl11 the default openssl devel package
```

Comment:
Can this be done using the `update-alternatives` command?

Reply:
I could not find such option

Reply:
I mean, its possible, but will require manual configuration which will end up with more setup than this 3 lines.

Reply:
update-alternatives is a debian based (i.e ubuntu, pop) specific thing.

Reply:
so you're saying line 10 doing nothing?